packages/piro/src/Piro_TempusSolver_Def.hpp

```diff
@@ -386,198 +385,109 @@ void Piro::TempusSolver<Scalar>::evalModelImpl(
     const Thyra::ModelEvaluatorBase::OutArgs<Scalar>& outArgs) const
 {
 #ifdef DEBUG_OUTPUT
-  *out << "DEBUG: " << __PRETTY_FUNCTION__ << "\n";
+  *out_ << "DEBUG: " << __PRETTY_FUNCTION__ << "\n";
 #endif
   using Teuchos::RCP;
   using Teuchos::rcp;
 
-  // TODO: Support more than 1 parameter and 1 response
-  const int j = 0;
-  const int l = 0;
-
-  // Parse InArgs
-  RCP<const Thyra::VectorBase<Scalar> > p_in;
-  if (num_p > 0) {
-    p_in = inArgs.get_p(l);
-  }
-  RCP<const Thyra::VectorBase<Scalar> > p_in2;  //JF add for multipoint
-  if (num_p > 1) {
-    p_in2 = inArgs.get_p(l+1);
-  }
-
-  // Parse OutArgs
-  RCP<Thyra::VectorBase<Scalar> > g_out;
-  if (num_g > 0) {
-    g_out = outArgs.get_g(j);
-  }
-  const RCP<Thyra::VectorBase<Scalar> > gx_out = outArgs.get_g(num_g);
-
-  Thyra::ModelEvaluatorBase::InArgs<Scalar> state_ic = model->getNominalValues();
-
-  // Set initial time in ME if needed
-
-  if(t_initial > 0.0 && state_ic.supports(Thyra::ModelEvaluatorBase::IN_ARG_t)) {
-    state_ic.set_t(t_initial);
+  // Set initial time and initial condition 
+  Thyra::ModelEvaluatorBase::InArgs<Scalar> state_ic = model_->getNominalValues();
+  if(t_initial_ > 0.0 && state_ic.supports(Thyra::ModelEvaluatorBase::IN_ARG_t)) {
+    state_ic.set_t(t_initial_);
   }
-
-  if (Teuchos::nonnull(initialConditionModel)) {
+  
+  if (Teuchos::nonnull(initialConditionModel_)) {
+    // IKT, 4/10/20, FIXME: I don't understand this case...
     // The initial condition depends on the parameter
     // It is found by querying the auxiliary model evaluator as the last response

```

Comment:
Did you figure out this comment? If not, maybe @etphipp can answer?

Reply:
Thanks for reminding me about this. We don't need any of the initialConditionModel_ stuff right now.  This is a remnant from Rythmos, and initial conditions for Tempus are set a different way.  The code with the initialConditionModel_ was never called anywhere.  Therefore I removed the code, hopefully removing future confusion as well.